Management/src/ActuatorWeb/BasicAuthExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System.Security.Claims;
+using idunno.Authentication.Basic;
+using Microsoft.AspNetCore.Authentication;
+
+namespace Steeltoe.Samples.ActuatorWeb;
+
+internal static class BasicAuthExtensions
+{
+    public static void ConfigureActuatorAuth(this IServiceCollection services)
+    {
+        AuthenticationBuilder builder = services.AddAuthentication(BasicAuthenticationDefaults.AuthenticationScheme);
+
+        builder.AddBasic(BasicAuthenticationDefaults.AuthenticationScheme, options =>
+        {
+            // This line allows basic auth to work over HTTP. See the note in the README under "Regarding HTTPS and Basic Authentication"
```

Comment:
This should just explain why we need this, so users can decide if they need it too. I'm not a fan of creating a dependency with a section title in docs, which may change.
Suggestion:

> // We need basic auth to work over HTTP, because docker containers don't trust the ASP.NET dev certificate.